src/GitHub.Exports/Settings/PkgCmdID.cs

```diff
@@ -18,5 +19,10 @@ public static class PkgCmdIDList
         public const int githubCommand = 0x320;
         public const int helpCommand = 0x321;
         public const int blameCommand = 0x500;
```

Comment:
This ID is referenced by `OpenPullRequests` but isn't actually defined in the `GitHub.VisualStudio.vsct` file. Probably worth fixing in a different PR.